vcd_cli/org.py

```diff
@@ -139,3 +143,40 @@ def create(ctx, name, full_name):
         stdout('Org \'%s\' is successfully created.' % result.get('name'), ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@org.command(short_help='delete an organization')
+@click.pass_context
+@click.argument('name',
+                metavar='<name>',
+                required=True)
+@click.option('-r',
+              '--recursive',
+              is_flag=True,
+              help='removes the Org and any objects it contains that are in a '
+                   'state that normally allows removal')
+@click.option('-f',
+              '--force',
+              is_flag=True,
+              help='pass this option along with --recursive to remove an Org'
```

Comment:
What happens if we pass --force without --recursive?

Reply:
It the org does not have orgVDC, it will force delete the org.  Otherwise, the command will fail as follows.
C:\sources\github\service_ecosystem\vcd-cli (master) 
(vcd-cli) λ vcd org delete Org2 --force
Are you sure you want to delete the Org? [y/N]: y
Do you want to force delete 'Org2' [y/N]: y
task: e218ba75-97c6-4c58-a1b2-07a54fc9be22, result: error, message: [ 173dea4e-5442-41c7-a296-a9f048f2353a ] You must delete this organization's VDCs before you can delete the organization